build.gradle.kts

```diff
@@ -17,6 +17,7 @@ allprojects {
 	repositories {
 		jcenter()
 		google()
+		maven { setUrl("https://jitpack.io") }
```

Comment:
Jitpack is a PITA to support, it often breaks and is slow. I'm glad we finally removed it when I put in the effort to migrate the (now deprecated) apiclient to JCenter. You'll need to find an alternative to this cause I don't want to depend on jitpack again.

Reply:
One solution for this is to copy the BlurHashDecoder class to our codebase and adding a license header, not the best solution but it doesn't look like the BlurHash lib is published anywhere